### PR TITLE
Added average video views.

### PR DESCRIPTION
**Pull request by dsjen**
screen shot 2015-09-02 at 6 07 09 pm

@mulby @jab5569 @brianhw

### CODE REVIEW

**Comment by dsjen** [comment]
I've updated with copy changes to chart tooltips.

File: analytics_dashboard/static/js/engagement-videos-main.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+/**
+ * Called for displaying a collection of video charts and tables.  Each bar represents a single video.
+ */
+require(['vendor/domReady!', 'load/init-page'], function (doc, page) {
+    'use strict';
+
+    require(['d3', 'underscore', 'views/data-table-view', 'views/stacked-bar-view'],
+        function (d3, _, DataTableView, StackedBarView) {
+            var model = page.models.courseModel,
+                graphSubmissionColumns = [
```
Comment (mulby) [inline analytics_dashboard/static/js/engagement-videos-main.js]: nit: replace "Submission" with "Video"

**Comment by mulby** [comment]
:+1:

File: analytics_dashboard/static/js/engagement-videos-main.js (hunk 1 above)
Comment (dsjen) [inline analytics_dashboard/static/js/engagement-videos-main.js]: Good catch!  Updated.

**Comment by dsjen** [comment]
Thanks, @mulby!  https://github.com/edx/edx-analytics-dashboard/pull/348 (js lint fix) is merged into master, and I've rebased this branch against it.  I'm waiting for tests to pass before merging in.

**Pull request merged**